.gitattributes

```diff
@@ -0,0 +1,12 @@
+# default to automatic recognition whether a file contains text:
+* text=auto
```

Comment:
This would actually be enough – the heuristics used by Git correctly classify all files in PP.